usaspending_api/templates/index.html

```diff
@@ -26,7 +26,7 @@
             <nav class="navbar">
                 <div class="container usa-da-header-container">
                     <h1>
-                      <a href="https://www.usaspending.gov"><img class="logo" src='{% static "img/logo.png" %}'/></a>
+                      <a href="https://www.usaspending.gov"><img class="logo" src='{% static "img/logo.png" %}' alt="Usaspending.gov"/></a>
```

Comment:
Any reason the 'U' in 'Usaspending.gov' is capitalized?

Reply:
No particular reason, would you prefer lowercase or USAspending.gov

Reply:
I think it would be better to keep the domain name lower case here